### DISCUSSION TITLE
Cant get started with EasyAdmin Bundle.

### DISCUSSION THREAD

**Discussion by chris576** [symfony/symfony#50847] [Q&A]
I have installed easy admin bundle with `composer require easycorp/easyadmin-bundle`.
But when I call `php bin/console make:admin:dashboard`, they tell me, that within make there is no admin.

**Comment by MatTheCat** [comment]
Maybe your PHP version is too old to install EasyAdminBundle ≥ 3? Which is it?

Also, what does `composer why easycorp/easyadmin-bundle` display?
